src/keysBuilder/TSExtractor.ts

```diff
@@ -1,34 +1,44 @@
+import { tsquery } from '@phenomnomnominal/tsquery';
 import { readFile } from '../helpers/readFile';
 import { regexs } from '../regexs';
 import { ExtractorConfig, ScopeMap, Scopes } from '../types';
-import { resolveScopeAlias } from './resolveScopeAlias';
 import { addKey } from './addKey';
 import { extractCommentsValues } from './commentsSectionExtractor';
-import { tsquery } from '@phenomnomnominal/tsquery';
+import { extractMarkerKeys } from './extractTSMarkerKeys';
 import { extractPureKeys } from './extractTSPureKeys';
 import { extractServiceKeys } from './extractTsServiceKeys';
+import { resolveScopeAlias } from './resolveScopeAlias';
 
 export function TSExtractor({ file, scopes, defaultValue, scopeToKeys }: ExtractorConfig): ScopeMap {
   const content = readFile(file);
-  if (!content.includes('@ngneat/transloco')) return scopeToKeys;
+  const extractors = [];
+  if (content.includes('@ngneat/transloco')) {
+    extractors.push(extractServiceKeys, extractPureKeys);
+  }
+  if (content.includes('@ngneat/transloco-keys-manager/marker') || content.includes('@ngneat/transloco-keys-manager')) {
```

Comment:
`includes('@ngneat/transloco-keys-manager/marker')` is a redundent check.
If `includes('@ngneat/transloco-keys-manager')` is ture than `includes('@ngneat/transloco-keys-manager/true')` is true as well.

Reply:
done